include/mapnik/expression_grammar_x3_def.hpp

```diff
@@ -70,6 +70,16 @@ namespace mapnik { namespace grammar {
     auto const& escaped_unicode = json::grammar::escaped_unicode;
     }
 
+    template <typename Context>
+    inline mapnik::transcoder const& extract_transcoder(Context const& ctx)
+    {
+#if BOOST_VERSION >= 106700
+        return x3::get<transcoder_tag>(ctx);
+#else
+        return x3::get<transcoder_tag>(ctx).get();
```

Comment:
this branch is unnecessary. `reference_wrapper` is implicitly convertible to the stored reference

Reply:
:+1: